packages/twenty-server/src/modules/workspace-member/standard-objects/workspace-member.workspace-entity.ts

```diff
@@ -94,6 +94,49 @@ export class WorkspaceMemberWorkspaceEntity extends BaseWorkspaceEntity {
   })
   userId: string;
 
+  @WorkspaceField({
+    standardId: WORKSPACE_MEMBER_STANDARD_FIELD_IDS.timeZone,
+    type: FieldMetadataType.TEXT,
+    label: 'Time zone',
+    defaultValue: "'system'",
+    description: 'User time zone',
+    icon: 'IconTimezone',
+  })
+  timeZone: string;
+
+  @WorkspaceField({
+    standardId: WORKSPACE_MEMBER_STANDARD_FIELD_IDS.dateFormat,
+    // type: FieldMetadataType.SELECT,
+    type: FieldMetadataType.TEXT,
+    label: 'Date format',
+    description: "User's preferred date format",
+    icon: 'IconCalendarEvent',
+    // options: [
+    //   { value: 'system', label: 'system', position: 0, color: 'turquoise'  },
+    //   { value: 'MMM_d_yyyy', label: 'MonthFirst', position: 1, color: 'red' },
+    //   { value: 'd_MMM_yyyy', label: 'DayFirst', position: 2, color: 'purple' },
+    //   { value: 'yyyy_MMM_d', label: 'YearFirst', position: 3, color: 'sky' },
+    // ],
+    defaultValue: "'system'"
```

Comment:
We can define enums for this field, however I didn't find any example in codebase where we define core workspace fields with enums as value? or maybe I missed something

Reply:
If I keep the type as `SELECT` and use options I get the following error

<img width="1167" alt="Screenshot 2024-06-02 at 9 50 37 PM" src="https://github.com/twentyhq/twenty/assets/13139771/2b6f7b5c-a78a-4511-be88-9fa6dbbac497">

Reply:
and you should use enum values instead of string values